libfwupdplugin/fu-hidraw-device.c

```diff
@@ -521,7 +521,7 @@ fu_hidraw_device_get_report(FuHidrawDevice *self,
 static void
 fu_hidraw_device_init(FuHidrawDevice *self)
 {
-	fu_udev_device_add_open_flag(FU_UDEV_DEVICE(self), FU_IO_CHANNEL_OPEN_FLAG_READ);
+	fu_udev_device_add_open_flag(FU_UDEV_DEVICE(self), FU_IO_CHANNEL_OPEN_FLAG_READ|FU_IO_CHANNEL_OPEN_FLAG_WRITE);
```

Comment:
you need to run clang-format on this -- did you call ./contrib/setup and set up a venv?